docs/anatomy/anatomy-releases.md

```diff
@@ -67,6 +67,27 @@ Follow the below steps to tag your Release:
 
 ![Confirm Tag](/img/anatomy/confirm-tag.png)
 
+### Promote a Release
+
+KintoHub provides you an option to `Promote` your Releases across the different environments.
+
+You can do the following to **Promote** the release:
+
+1. Click on the `...` in the row of the `Tagged` release you wish to promote.
+2. Click on the **Promote Tag to Env** option
+
+![Promote Env](/img/anatomy/promote-env.png)
+
+3. Select the environment from the list you wish to promote the release in the prompt displayed.
+4. Click on the **Promote Release** button.
+
+![Confirm Promote](/img/anatomy/confirm-promote.png)
+
+:::note
+You cannot promote the release which is not tagged and the `Promote Tag to Env` option is disabled for the untagged releases.
```

Comment:
typo : promote a release.
Actions : make the note in 2 points? 
1. You cannot promote a release which is not tagged.
2. the `Promote Tag to Env` option is disabled for untagged releases.

Reply:
Done